src/pytest_stepfunctions/services/aws_lambda.py

```diff
@@ -3,9 +3,20 @@
 import re
 import sys
 import traceback
-from typing import Match, Optional
-from urllib.parse import urlparse, ParseResult
+from dataclasses import dataclass
 from http.server import BaseHTTPRequestHandler
+from typing import Match, Optional
+from urllib.parse import ParseResult, urlparse
+
+
+@dataclass
+class LambdaContext:
+    function_name: str = "test-function"
+    memory_limit_in_mb: int = 128
+    invoked_function_arn: str = (
+        f"arn:aws:lambda:us-east-1:123456789012:function{function_name}"
+    )
+    aws_request_id: str = "da658bd3-2d6f-4e7b-8ec2-937234644fdc"
```

Comment:
This could probably be changed to `aws_request_id: str = str(uuid4())` to dynamically generate a request id.